### PR TITLE
Pin Colors Dep - Hard Error

### PR DESCRIPTION
**Pull request by ingalls**
### Context

`colors` package issue: https://www.theverge.com/2022/1/9/22874949/developer-corrupts-open-source-libraries-projects-affected

### CODE REVIEW

**Comment by bilindhajer** [comment]
> ### Context
> `colors` package issue: https://www.theverge.com/2022/1/9/22874949/developer-corrupts-open-source-libraries-projects-affected

How do you plan to roll this out? i.e. minor or patch? Also how did you test this? Please leave notes if possible.

**Comment by ingalls** [comment]
> How do you plan to roll this out? i.e. minor or patch?

I think it is safe to deploy this via a patch, it should have no breaking changes or downstream users provided they are running on > node 10.

> Also how did you test this? 
- Run `npm install` or `npm upgrade` on this repo or a parent repo using this as a library.
- `npm ls colors` will then list `colors@1.4.2` (Affected Version)
- Subsequent runs of any command that uses colors - IE `info` will result in an infinite loop of garbage populating the screen.

I've reproduced the bug in the parent library `openaddresses/deploy` and confirmed the fix present in this branch in https://github.com/openaddresses/deploy/pull/27

File: package.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
+    "colors": "1.4.0",
     "cuid": "^2.1.1",
     "d3-queue": "^3.0.2",
     "diff": "^3.5.0",
     "easy-table": "^1.0.0",
     "fasterror": "^1.0.0",
-    "inquirer": "^6.1.0",
-    "json-diff": "^0.5.2",
+    "inquirer": "^8.2.0",
+    "json-diff": "^0.7.1",
```
Comment (bilindhajer) [inline package.json:23]: Can we keep this PR exclusive to colors and have separate ones to update other packages. Or did you do this for another reason?

**Review by bilindhajer — CHANGES_REQUESTED** [review]

**Comment by bilindhajer** [comment]
> > How do you plan to roll this out? i.e. minor or patch?
> 
> I think it is safe to deploy this via a patch, it should have no breaking changes or downstream users provided they are running on > node 10.
> 
> > Also how did you test this?
> 
> * Run `npm install` or `npm upgrade` on this repo or a parent repo using this as a library.
> * `npm ls colors` will then list `colors@1.4.2` (Affected Version)
> * Subsequent runs of any command that uses colors - IE `info` will result in an infinite loop of garbage populating the screen.
> 
> I've reproduced the bug in the parent library `openaddresses/deploy` and confirmed the fix present in this branch in [openaddresses/deploy#28](https://github.com/openaddresses/deploy/pull/28)

Ok sounds good. I left a question on your PR.

File: package.json (hunk 1 above)
Comment (ingalls) [reply package.json:23]: It's been historically difficult to get things merged into this repo & npm audit was throwing a warning for both of these. An upstream client of DevelopmentSeed's had problems with their security compliance department due to the npm audit flag and as such I had bundled them together.

Comment (bilindhajer) [reply package.json:23]: ok, for now I think it is fine, if we have any issues we might roll back.

**Review by bilindhajer — APPROVED** [review]
Approved, and merged. We will tag/publish.

**Pull request merged**